content/docs/concepts/multicluster-deployments/index.md

```diff
@@ -12,26 +12,67 @@ a single controllable composite application.
+However, for many reasons such as scale, redundancy, workload isolation, compliance, integration with legacy services,
+etc., most applications will eventually need to be distributed and have some of their services running elsewhere.
 
 Istio supports many possible topologies for distributing the services of an
 application beyond a single cluster, for example:
 
-* Services within the mesh can use [service entries](/docs/concepts/traffic-management/#service-entries)
+* You can use [egress traffic control](/docs/tasks/traffic-management/egress/) to allow workloads inside the mesh
   to access standalone external services or to access services exposed by another loosely-coupled service mesh.
 * You can [expand the service mesh](/docs/setup/kubernetes/additional-setup/mesh-expansion/) to include
   services running on VMs or bare metal hosts.
 * You can combine the services from more than one cluster into a single composite service mesh.
+* You can combine autonomous service meshes in multiple clusters into a loosely coupled [federation of service meshes](#mesh-federation).
+
+## Criteria for selecting the right topology
+
+When you plan your multicluster topology, you first need to decide if you are going to combine the clusters into a
+single multicluster mesh or create a federation of multiple meshes. Ask yourself the following questions:
+
+**Uniform naming**: Will your meshes have _uniform naming_. In case of multiple Kubernetes clusters, _uniform naming_
+means that a service named `foo` in namespace `ns1` of cluster 1 has the same API and semantics as a service named `foo`
+in `ns1` of cluster 2.
+
+**Exposing behavior**: Will your combined clusters expose all their services one to another, or selective exposure is
+required? _Selective exposure_ means that the owner of cluster 1 may want to control which services are exposed to the
+workloads in cluster 2. Selective exposure may be required for security or compliance reasons. Some security standards
+require exposure of services on the need-to-know basis.
+
+**Isolation for security or compliance**: Are you required to control all the traffic that enters your cluster **at the
+point of entrance**, for example at the Ingres Gateway?
+
+**Common trust**: Can you setup a common CA root for your clusters? Will you perform authentication by Istio
+[mutual TLS](/docs/concepts/security/#mutual-tls-authentication) or will you use access tokens like
+[JWT](https://istio.io/docs/concepts/security/#origin-authentication)?
+
+**Operating multiple control planes**: Do you have infrastructure capabilities to manage multiple Istio control planes
+or do you prefer to manage a single control plane for multiple clusters?
+
+**Providing access to Kubernetes APIs**: Are you allowed to provide access to Kubernetes APIs of one cluster to
+other clusters?
+
+**Sending telemetry to external meshes**: Are you allowed to send telemetry data from your cluster to other clusters, from
```

Comment:
@frankbu proposed to stick to clusters instead of meshes. The question is not "does your cluster send telemetry data to another cluster?". The Shared Control Plane implementation of Istio requires this sending. The question is: "are you OK with that"? From security, compliance, procedures point of view.

The second question is are you OK that your pods access the other cluster directly, without going through some egress gateways in your clusters? In some environments, the outgoing traffic is allowed only through specific pods.